tests/testthat/test-add_highlight.R

```diff
@@ -373,25 +372,26 @@ testthat::test_that("T4.2 The function also reduces the alpha value of the confi
   gg_CI_fills_numeric <- gsub("#[A-Z0-9]{6}", "", gg_CI_fills) %>%
     sapply(function(s) {
 
-      visR:::convert_alpha(hex_alpha = s)
+      visR:::.convert_alpha(hex_alpha = s)
```

Comment:
::: is actually not a recommended practice. If we cant use .convert_alpha is test then we should make it an exported